### PR TITLE
Detector frame masses

### PR DESCRIPTION
**Pull request by u-dupletsa**
- added test for checking that SNR output does not change with different mass parametrizations
- fixed input in tutorial according to new mass parametrization
- fixed bug in waveforms.py: canceled the redundant (1+z) factor

### CODE REVIEW

File: GWFish/modules/waveforms.py [hunk 1]
```diff
@@ -286,8 +286,8 @@ def _setup_lal_caller_args(self):
             self._waveform_postprocessing = self._hf_postproccessing_SimInspiralFD
             self._lalsim_caller = lalsim.SimInspiralFD
             self._lalsim_args = [
-                self.gw_params['mass_1'] * lal.MSUN_SI * (1 + self.gw_params['redshift']),  # in [kg]
-                self.gw_params['mass_2'] * lal.MSUN_SI * (1 + self.gw_params['redshift']),  # in [kg]
+                self.gw_params['mass_1'] * lal.MSUN_SI,# * (1 + self.gw_params['redshift']),  # in [kg]
```
Comment (jacopok) [inline GWFish/modules/waveforms.py:289]: I think we can fully remove it, not just comment it (less confusion for people who come later)

Comment (u-dupletsa) [reply GWFish/modules/waveforms.py]: Done!

File: tests/test_masses.py [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+
+def test_snr_different_mass_inputs():
+
+    detectors = ['ET']
+    network = Network(detector_ids = detectors, detection_SNR = (0., 8.))
+    waveform_model = 'IMRPhenomD_NRTidalv2'
+
+    z = np.array([0.00980])
+
+    parameters_source = {
```
Comment (jacopok) [inline tests/test_masses.py:27]: Really nice! 

I would just propose a little cleanup: you can define a dictionary of `extra params` (ra, dec...) just once, and then each time combine them like

```
params_det = {masses_det} | extra_params
```
and similarly for the others, so you only change the masses

Comment (jacopok) [reply tests/test_masses.py:27]: and it makes the function more readable

Comment (u-dupletsa) [reply tests/test_masses.py:27]: Nice, on it!

**Review by jacopok — APPROVED** [review]

**Pull request merged**